config/samples/workloads_v1alpha1_consoletemplate.yaml

```diff
@@ -22,6 +22,7 @@ spec:
       restartPolicy: Never
 metadata:
   name: console-template-0
+  namespace: default
```

Comment:
Why add namespace here?

Reply:
`namespace: default` is redundant, but in practice we will add namespaces to ConsoleTemplates, which the Console will inherit.

Reply:
I still don't see that it adds any value in this example - this is obviously a namespaced resource so the object will have to go in *some* namespace. Adding this here just introduces the downside that if you were to pipe this through `kubectl apply` while targeting a different namespace in your current config, then the resource isn't going to go where you'd expect.

Reply:
> if you were to pipe this through `kubectl apply` while targeting a different namespace in your current config, then the resource isn't going to go where you'd expect.

I was about to reply with "That isn't true" but it depends on what you mean by "expect"? I don't expect kubectl apply to only modify resources in my namespace, it depends on what's in the resource definitions.

In any case I'm not too fussed, I'll remove the namespace.